### PR TITLE
Implement the AddEntry view

### PR DESCRIPTION
**Pull request by Gallost**

### CODE REVIEW

File: app/src/main/java/com/example/cityspots/view/AddEntryScreen.kt [hunk 1]
```diff
@@ -1,40 +1,302 @@
...
+                        )
+                        navController.popBackStack()
+                    }) {
+                        Text("Add Entry")
+                    }
+                }
+            }
+        }
+    }
+//    Spacer(modifier = Modifier.height(8.dp))
```
Comment (adrian-salvador) [inline app/src/main/java/com/example/cityspots/view/AddEntryScreen.kt]: is this commented section needed in the final version?

**Review by dharinirajah — COMMENTED** [review]
slay

**Review by dharinirajah — APPROVED** [review]
I forgot to hit approve, but still slay

**Review by adrian-salvador — APPROVED** [review]
besides my comment above, it looks good 👍

**Pull request merged**